aldryn_search/views.py

```diff
@@ -63,6 +63,7 @@ def get_queryset(self):
         queryset = self.form.search()
         if not self.request.user.is_authenticated():
             queryset = queryset.exclude(login_required=True)
+        queryset = queryset.exclude(url__in=['', None])
```

Comment:
I don't agree with this because it assumes the dev will have a url field with values.